### PR TITLE
docs: update modifier docs

### PR DESCRIPTION
**Pull request by okke-formsma**
Some improvements on the modifier docs.

### CODE REVIEW

File: docs/docs/codes/modifiers.mdx [hunk 1]
```diff
@@ -30,13 +30,15 @@ These act like any other key code.
...
-
-- They _apply_ the modifier to another _code_.
-  - e.g. `&kp LS(A)` = `LEFT_SHIFT`+`A`
-    - aka a _shifted_ (capitalized) **A**.
-  - The _code_ and the _modifier_ are handled _as one_.
-- They can be _combined_:
-  - e.g. `&kp LC(RA(B))` = `LEFT_CONTROL`+`RIGHT_ALT`+`B`
-- Some codes _include_ modifiers in their definition.
-  - e.g. `DOLLAR` = `LS(N4)`
+Modifier functions add one or more modifier to another code. These functions take the form: `XX(code)`
```
Comment (innovaker) [inline docs/docs/codes/modifiers.mdx:33]: Modifier functions add one or more modifier to a ~~another~~ code.

Comment (innovaker) [inline docs/docs/codes/modifiers.mdx:33]: > Modifier functions add one or more modifier to another code.  These functions take the form: `XX(code)`

Recommend a new line between those sentences for readability.

File: docs/docs/codes/modifiers.mdx [hunk 2]
```diff
@@ -30,13 +30,15 @@ These act like any other key code.
...
+
+- Modifier functions apply the modifier to another code:
+  - `&kp LS(A)` = `LEFT_SHIFT`+`A` (a capitalized **A**).
+- They can be combined:
+  - `&kp LC(RA(B))` = `LEFT_CONTROL`+`RIGHT_ALT`+`B`
+- Some basic codes codes already include a modifier function in their definition:
+  - `DOLLAR` = `LS(NUMBER_4)`
+- There are left- and right-handed versions of each modifier:
```
Comment (innovaker) [inline docs/docs/codes/modifiers.mdx:41]: Do we need this with the table above?

File: docs/docs/codes/modifiers.mdx [hunk 3]
```diff
@@ -30,13 +30,15 @@ These act like any other key code.
...
+  - `LS`, `LC`, `LA`, `LG`, `RS`, `RC`, `RA`, `RG`
+- Modified keys can safely be rolled-over. Modifier functions are released when another key is pressed.
```
Comment (innovaker) [inline docs/docs/codes/modifiers.mdx:43]: What does "rolled-over" mean?  If I don't understand that, I doubt a beginner will.

File: docs/docs/codes/modifiers.mdx [hunk 4]
```diff
@@ -30,13 +30,15 @@ These act like any other key code.
...
+  - Press `&kp LS(A)`, then press `&kp B`, release `&kp A` and release `&kp B` results in **Ab**. Only the A is capitalized.
```
Comment (innovaker) [inline docs/docs/codes/modifiers.mdx:44]: For me, this wasn't easy to understand.  I had to really think about it.  I realise it's not easy to convey.

**Review by innovaker — CHANGES_REQUESTED** [review]
Looks alright.  Just some suggestions.

As discussed on [Discord](https://discord.com/channels/719497620560543766/719544230497878116/774735273568501761), I also recommend you drop the "another" in "another code".

File: docs/docs/codes/modifiers.mdx (hunk 2 above)
Comment (okke-formsma) [reply docs/docs/codes/modifiers.mdx:41]: I think it's nice to have them in one place. I immediately thought "ok which modifier functions exist?" when I read this chapter in the docs.

File: docs/docs/codes/modifiers.mdx (hunk 3 above)
Comment (okke-formsma) [reply docs/docs/codes/modifiers.mdx:43]: that's why there's an example. It's a common phrase in keyboard land.

File: docs/docs/codes/modifiers.mdx (hunk 1 above)
Comment (okke-formsma) [reply docs/docs/codes/modifiers.mdx:33]: ok

Comment (okke-formsma) [reply docs/docs/codes/modifiers.mdx:33]: done

File: docs/docs/codes/modifiers.mdx (hunk 2 above)
Comment (innovaker) [reply docs/docs/codes/modifiers.mdx:41]: Perhaps add a note to see the table above as it also provides more information around each modifier.

File: docs/docs/codes/modifiers.mdx (hunk 3 above)
Comment (innovaker) [reply docs/docs/codes/modifiers.mdx:43]: I still didn't understand with the example, but I'll trust your judgement.

File: docs/docs/codes/modifiers.mdx [hunk 5]
```diff
@@ -30,13 +30,17 @@ These act like any other key code.
 
 ### Modifier Functions
 
+Modifier functions add one or more modifier to another code.
```
Comment (innovaker) [inline docs/docs/codes/modifiers.mdx:33]: ~~another~~ code

File: docs/docs/codes/modifiers.mdx [hunk 6]
```diff
@@ -30,13 +30,17 @@ These act like any other key code.
 
 ### Modifier Functions
 
+Modifier functions add one or more modifiers to a keycode.
```
Comment (innovaker) [inline docs/docs/codes/modifiers.mdx:33]: ~~key~~code
? - to match with everything else.

Incidentally that's also why I had that word italicized originally.

File: docs/docs/codes/modifiers.mdx (hunk 4 above)
Comment (okke-formsma) [reply docs/docs/codes/modifiers.mdx]: I don't know how to write this down more clearly. "rolling over" is established keyboard-speak, and the next sentence and the example both explain in more detail.

File: docs/docs/codes/modifiers.mdx [hunk 7]
```diff
@@ -30,13 +30,17 @@ These act like any other key code.
 
 ### Modifier Functions
 
+Modifier functions add one or more modifiers to a code.
```
Comment (innovaker) [inline docs/docs/codes/modifiers.mdx:33]: I just realised this sentence duplicates the first bullet point, but up to you whether you want to fix that.

Comment (okke-formsma) [reply docs/docs/codes/modifiers.mdx:33]: I think its fine to repeat the description like this. I see the list as describing each of the features of this functionality in more detail with examples.

File: docs/docs/codes/modifiers.mdx [hunk 8]
```diff
@@ -30,13 +30,17 @@ These act like any other key code.
...
-  - The _code_ and the _modifier_ are handled _as one_.
-- They can be _combined_:
-  - e.g. `&kp LC(RA(B))` = `LEFT_CONTROL`+`RIGHT_ALT`+`B`
-- Some codes _include_ modifiers in their definition.
-  - e.g. `DOLLAR` = `LS(N4)`
+- Modifier functions apply a modifier to a code:
+  - `&kp LS(A)` = `LEFT_SHIFT`+`A` (a capitalized **A**).
+- They can be combined:
+  - `&kp LC(RA(B))` = `LEFT_CONTROL`+`RIGHT_ALT`+`B`
+- Some basic codes codes already include a modifier function in their definition:
```
Comment (innovaker) [inline docs/docs/codes/modifiers.mdx:41]:
```suggestion
- Some basic codes already include a modifier function in their definition:
```

Comment (petejohanson) [reply docs/docs/codes/modifiers.mdx:41]: @okke-formsma this PR looks good, once this suggested fix is in. Thanks!

**Review by petejohanson — APPROVED** [review]

**Pull request merged**